### ISSUE TITLE
Limitations on filters to 10 records?

### ISSUE THREAD

**Issue by NorthFred** [mevdschee/php-crud-api#701]
I have a list call with 13 filters applied. I noticed that three items were missing all the time, and when playing around with filters in the url, I noticed that there appears to be a limitation of 10 returned records. The filters after the 10th index are ignored by the back-end.

I checked the documentation, but there is no information regarding that.  Can this be overridden, and if so, how? Or is 10 the absolute maximum that can be set?

Example of request:
```
GET /api/api.php/records/Items?filter=status,eq,1&order=name,asc&filter0=id,eq,101&filter1=id,eq,132&filter2=id,eq,155&........&filter12=id,eq,219
```

**Comment by mevdschee** [comment]
>Or is 10 the absolute maximum that can be set?

Not anymore.. but you need to use letters when you run out of numbers.. Like this: 0,1,2,3,4,5,6,7,8,9,a,b,c,d,e,f,...

**Comment by mevdschee** [comment]
I released: v2.9.21 Allow more than 10 filters
